calcite/Calcite/CheckIndicator.qml

```diff
@@ -21,10 +21,10 @@ Rectangle {
     height: 24
     border {
         width: 2
-        color: control.checkState !== Qt.Unchecked ? control.hovered ? Calcite.brandHover : Calcite.brand :
-        control.hovered ? Calcite.text3 : Calcite.borderInput
+        color: control.checkState !== Qt.Unchecked ? control.hovered && control.enabled ? Calcite.brandHover : Calcite.brand :
```

Comment:
Just putting my thoughts out there but this ternary logic is starting to get out of hand but I see this isn't new as I see them all other the place in this Component. Nothing to change unless you feel otherwise.

Reply:
maybe you could combine `control.hovered && control.enabled` into a single property and use that in place since that one is repeated quite a bit? not sure how that works with property binding but just an idea to reduce the length of these.